Facade_Engine/Query/PrimaryPropertyName.cs

```diff
@@ -90,7 +93,7 @@ public static string PrimaryPropertyName(this IEdge edge)
         [Output("propName", "The name of the element's construction property")]
         public static string PrimaryPropertyName(this Panel panel)
         {
-            if (panel.Construction == null)
+            if (panel == null || panel.Construction == null)
```

Comment:
Same thing here.

Reply:
This was set up as was, so figured the discipline were happy with an empty string with no warning. If warnings are desired, I am leaving that to the discipline to add either to this PR before merge, or after in their own PR :smile: